Makefile

```diff
@@ -15,7 +15,7 @@ include $(addprefix ./vendor/github.com/openshift/build-machinery-go/make/, \
 #
 # Example:
 #   make check
-check: | verify test-unit
+check: | verify verify-bindata test-unit
```

Comment:
Just to make sure -- CI will run `check` which means that bindata will also be verified, and in case there's a diff between the api copy and the local copy, we'll get a failed test, right?

Reply:
Looking at https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-authentication-operator/openshift-cluster-authentication-operator-master.yaml I don't actually think `check` is run in CI explicitly, but I added this here since it seemed like a natural fit.

I'll plan to add a new CI check to run this make target today (although it will fail until this PR is merged).

Reply:
Created https://github.com/openshift/release/pull/61456 to add this as an explicit CI check